backend/src/main/java/com/fellas/bespoke/persistence/model/Activity.java

```diff
@@ -0,0 +1,47 @@
+package com.fellas.bespoke.persistence.model;
+
```

Comment:
This class definitely is not the class that we want to keep in the database. It is almost impossible to find a activity which is related to a specific topic or user. We should rethink how to keep things in database

Reply:
Yes, this is just for that week demonstration. We are gonna improve the functionality.